backend/processing/bundle.bash

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
As discussed, we should probably reuse the existing bundle.bash script. It should be pretty easy if you move it up a level and use `basename`/`dirname` on `$1`.

Reply:
Since 49287826c0b951828565effd6d92cb237f139e08 we use single Lambda bundling script